play/style.go

```diff
@@ -8,6 +8,10 @@ var (
 	answerStyle         = lipgloss.NewStyle().
 				PaddingLeft(1).PaddingRight(1).
 				BorderStyle(lipgloss.NormalBorder())
+	infoMessageStyle = lipgloss.NewStyle().
```

Comment:
Let's rename and use this colours. I just experimented with ascii colours. And it looks better for me

```
	warningMessageStyle = lipgloss.NewStyle().
				Foreground(lipgloss.Color("231")).
				Background(lipgloss.Color("166")).
				Padding(0, 1)
```

Reply:
I use this ansi colours table to match codes

https://github.com/fidian/ansi?tab=readme-ov-file#--color-codes